### PR TITLE
added schema for pluto19v2

### PR DESCRIPTION
**Pull request by sraby**

### CODE REVIEW

**Comment by sraby** [comment]
@toolness @austensen @0xStarcat @aepyornis, let me know if it's ok to merge this in! Nothing too fancy

File: src/nycdb/verify.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
     'pluto_18v1': {'pluto_18v1': 800_000},
     'pluto_18v2': {'pluto_18v2': 800_000},
     'pluto_19v1': {'pluto_19v1': 800_000},
+    'pluto_19v1': {'pluto_19v2': 800_000},
```
Comment (aepyornis) [inline src/nycdb/verify.py]: typo -- should be pluto_19v2

Comment (sraby) [reply src/nycdb/verify.py]: oops! ok just fixed, thanks for catching

**Comment by aepyornis** [comment]
Except for that small typo, it's all good to merge. Thank you!!

**Comment by sraby** [comment]
awesome, thanks @aepyornis

**Pull request merged**